scripts/build-a-core/template.py

```diff
@@ -34,7 +34,7 @@ def dockerfile_template(
 
 WORKDIR /opt/wasi-sdk
 ARG WASI_SDK_REF={wasi_sdk_ref}
-RUN git clone https://github.com/WebAssembly/wasi-sdk.git . && git checkout $WASI_SDK_REF && git submodule update --init --depth 10
+RUN git clone https://github.com/WebAssembly/wasi-sdk.git . && git checkout $WASI_SDK_REF && git submodule update --init --depth 50
```

Comment:
Why we need it increased?

Reply:
Submodule dependencies are (apparently) checked out by first cloning the repository and then checking out the commit the submodule points at.

One of the second-level submodule dependencies added more commits into its default branch so the top 10 did not include the commit that was supposed to get checked out, so we need to clone more commits to get the latest commit in there. Worst case scenario we need to clone the whole repo.

Reply:
I see, thanks for clarification